picasso/src/test/java/com/squareup/picasso/AssetRequestHandlerTest.java

```diff
@@ -8,11 +8,14 @@
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
 
+import static android.os.Build.VERSION_CODES.M;
 import static com.google.common.truth.Truth.assertThat;
 import static org.mockito.MockitoAnnotations.initMocks;
 
 @RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = M)
```

Comment:
You don't need the BuildConfig class when using AGP 3.0.0

Reply:
Plus we're about to stop generating it...